### ISSUE TITLE
[Bug] Injection broken after Canary 123373 c05ebe3 update

### ISSUE THREAD

**Issue by alsoGAMER** [BetterDiscord/BetterDiscord#1265]
**Describe the Bug**
Injection broken after Canary 123373 c05ebe3 update

**To Reproduce**
- Update to the latest canary version
- Reinject BD

**Expected Behavior**
BD should inject fine

**Discord Version**
Canary

**Comment by 1011025m** [comment]
Can confirm on my installation of canary

Canary 123373 (c05ebe3) 
Host 1.0.45 
Windows 10 64-Bit (10.0.19044)

**Comment by NextdoorPsycho** [comment]
Confirm on my canary as well:
Canary 123387 (14782d3) 
Host 1.0.45 
Windows 10 64-Bit (10.0.22000)

**Comment by OodavidsinoO** [comment]
Same here
Canary 123387 (14782d3)
Host 1.0.45
Windows 11 64-Bit (22000.593)

**Comment by sis-15** [comment]
Same goes here
Canary 123387 (14782d3)
Host 1.0.45 Windows 10 
64-Bit (10.0.22000)

**Comment by borntohonk** [comment]
Same here:
Canary 123387 (14782d3)
Host 1.0.45
Windows 11 64-Bit (10.0.22000)

**Comment by ZandathraX** [comment]
Same here

Canary 123387 (14782d3)
Host 1.0.45
Windows 11 64-Bit (10.0.22000)

**Comment by AlphaKR93** [comment]
Same here

Canary 123387 (14782d3)
Host 1.0.45
Windows 11 64-Bit (10.0.22593)

**Comment by ST3VI3-RICHI3** [comment]
Same here

Canary 123387 (14782d3)
Host 1.0.45
Windows 10 64-Bit (10.0.19044)

**Comment by SrIzan10** [comment]
same
```
Canary 123387 (14782d3)
Host 1.0.45
Windows 10 64-Bit (10.0.19043)
```

**Comment by Strencher** [comment]
It's not very useful to post the same thing over and over again. Just reacting with a 👍 to the initial message is enough.

**Comment by alsoGAMER** [comment]
> It's not very useful to post the same thing over and over again. Just reacting with a 👍 to the initial message is enough.

You're right, i was going to say that now.

**Comment by alsoGAMER** [comment]
Fixed via #9ee2ee6.
